extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,510 @@
+<!-- omit in toc -->
+# EXT_feature_metadata
+
+**Version 1.0.0** [TODO: date]
+
+<!-- omit in toc -->
+## Contributors
+
+* Peter Gagliardi, Cesium
+* Sean Lilley, Cesium
+* Bao Tran, Cesium
+* Sam Suhag, Cesium
+* Samuel Vargas, Cesium
+* Patrick Cozzi, Cesium
+
+<!-- omit in toc -->
+## Status
+
+Draft
+
+<!-- omit in toc -->
+## Dependencies
+
+Written against the glTF 2.0 specification.
+
+Optionally extends the [`EXT_mesh_gpu_instancing` extension](../../EXT_mesh_gpu_instancing/README.md).
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is optional, meaning it should be placed in the `extensionsUsed` list, but not in the `extensionsRequired` list.
+
+<!-- omit in toc -->
+## Contents
+
+- [Overview](#overview)
+- [Feature Identification](#feature-identification)
+  - [Feature ID Vertex Attributes](#feature-id-vertex-attributes)
+    - [Feature ID Accessors](#feature-id-accessors)
+    - [Implicit Feature IDs](#implicit-feature-ids)
+  - [Feature ID Textures](#feature-id-textures)
+  - [Feature ID Instance Attributes](#feature-id-instance-attributes)
+- [Feature Metadata](#feature-metadata)
+  - [Schemas](#schemas)
+  - [Feature Tables](#feature-tables)
+  - [Feature Textures](#feature-textures)
+  - [Statistics](#statistics)
+- [Examples](#examples)
+
+## Overview
+
+This extension allows offline batching of heterogeneous 3D features, such as different buildings in a city, for efficient streaming to a client for rendering and interaction. Efficiency comes from transferring multiple features in a single request and rendering them in the least number of draw calls necessary.
+
+Feature IDs enable individual features to be identified and updated at runtime, e.g., show/hide, highlight color, etc. Feature IDs may be assigned on a per-vertex, per-texel, or per-instance basis.
+
+Feature IDs may be used to access metadata, such as passing a building's ID to get its address. Per-feature metadata is stored in a compact binary tabular format described in the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/README.md).
+
+![Building Example](figures/feature-table-buildings.png)
+
+> In the diagram above, a glTF consists of two houses batched together into a single primitive. A feature ID attribute on the primitive indicates that all of the vertices making up the first house have a feature ID of 0, while all vertices making up the second house have the feature ID 1. The feature ID is then used to access the building's metadata from the feature table.
```

Comment:
just wondering, why is this a quote block? do we have other examples of using this for explanations?